src/main/java/model/DAO/OrderDAOImpl.java

```diff
@@ -0,0 +1,157 @@
+            "WHERE USER_ID = ?";
+
+    private final String GET_ORDER_BY_CAR_ID = "SELECT ORDER_ID, ORDER_DATE, USER_ID, TOTAL_COST, DISCOUNT " +
+            "FROM ORDER AND SELECT ORDER_DETAIL.ORDER_ID, " +
+            "ORDER_DETAIL.CAR_ID, ORDER_DETAIL.TARIF FROM ORDER_ID " +
+            "LEFT JOIN ORDER_DETAIL ON ORDER = ORDER_DETAIL.ORDER_ID " +
+            "LEFT JOIN ORDER_DETAIL ON CAR = ORDER_DETAIL.CAR_ID " +
+            "WHERE USER_ID = ?";
+
+    private final String ADD_ORDER = "INSERT INTO ORDER ORDER_DATE = ?, USER_ID = ?, TOTAL_COST = ?, DISCOUNT = ?" +
+            "INSERT INTO ORDER_DETAIL ORDER_ID = ?, CAR_ID = ?, TARIF = ?";
+
+    private final String UPDATE_ORDER = "UPDATE ORDER SET ORDER_ID = ?, ORDER_DATE = ?, USER_ID = ?, TOTAL_COST = ?, DISCOUNT = ? " +
+            "UPDATE ORDER_DETAIL SET ORDER_DETAIL_ID = ?, ORDER_ID = ?, CAR_ID = ?, TARIF = ? " +
+            "WHERE ORDER_ID = ?";
+
+    private final String DELETE_ORDER = "DELETE FROM ORDER WHERE ORDER_ID = ?";
+
+    private ConnectionPool connectionPool = ConnectionPool.getInstance();
+    private Connection connection = connectionPool.getConnection();
+
+    private Order collectingOrder(String sql, Order order){
+        try(PreparedStatement preparedStatement = connection.prepareStatement(sql);
+        ResultSet resultSet = preparedStatement.executeQuery()){
+            while(resultSet.next()){
+                order.setOrderID(resultSet.getInt("ORDER_ID"));
+                order.setOrderDate(resultSet.getDate("ORDER_DATE"));
+                order.getUserID().setUserID(resultSet.getInt("USER_ID"));
+                order.setTotalCost(resultSet.getInt("TOTAL_COST"));
+                order.setDiscount(resultSet.getInt("DISCOUNT"));
+                order.getCarID().setCarID(resultSet.getInt("CAR_ID"));
+                order.setTarif(resultSet.getInt("TARIF"));
+            }
+        } catch(SQLException e){
+            e.printStackTrace();
+        }
+        connectionPool.putBackConnection(connection);
+        return order;
+    }
+
+    private void creatingOrder(String sql, Order order) {
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+            preparedStatement.setInt(1, order.getOrderID());
+            preparedStatement.setDate(2, (Date) order.getOrderDate());
+            preparedStatement.setObject(3, order.getUserID().getUserID());
+            preparedStatement.setInt(4, order.getTotalCost());
+            preparedStatement.setInt(5, order.getDiscount());
+            preparedStatement.setObject(6, order.getCarID().getCarID());
+            preparedStatement.setInt(7, order.getTarif());
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        connectionPool.putBackConnection(connection);
+    }
+
+    public LinkedBlockingQueue<Order> getAllOrders(){
```

Comment:
rename getAllOrders to getAll, move getAll to AbstractDAO like List<T> getAll(); since the method is common and could be used in other DAOs